docs/experimental-features.md

```diff
@@ -53,7 +53,7 @@ run your daemon with the `--enable-pubsub-experiment` flag. Then use the
 
 Gossipsub is a new, experimental routing protocol for pubsub that
 should waste less bandwidth than floodsub, the current pubsub
-protocol. It's backwards compatible with floodsub so enabling this
```

Comment:
note: british english (but normalizing is fine)

Reply:
yeah I know, should have added this to the changelog